ui/index.ts

```diff
@@ -253,6 +253,12 @@ export {
     type RadioButtonGroupProps,
 } from './elements/Form/RadioButtonGroup';
 
+// NSIconGroup
+export {
+    type IconGroupProps,
+    default as NSIconGroup,
```

Comment:
lets create a test file for this new component, you can take the file `Button.test.tsx` for reference here. Lets think about the features/behaviour of this component (e.g., rendering or not, if correct icons are rendered or not, if color is applied correctly to all icons, etc.)